Panic when asking for version without being logged in

When asking for CDS version before being logged in to the API, I expect an output showing me the version of the cdsctl binary, and an error message telling me I need to log before asking for the API's version.

But the command crashes without helpful message.

To reproduce:
$ rm ~/.cdsrc
$ ./cdsctl version
Output:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xad465e]

goroutine 1 [running]:
main.versionRun(0xc0003932c0, 0xcb0f9a, 0x6)
	/tmp/QnVpbGQgYW5kIFBhY2thZ2UgQWxs/run/cli/cdsctl/version.go:53 +0x24e
github.com/ovh/cds/cli.newCommand.func2(0xc000381d40, 0x1386488, 0x0, 0x0)
	/tmp/QnVpbGQgYW5kIFBhY2thZ2UgQWxs/run/cli/cobra.go:250 +0x1d3f
github.com/spf13/cobra.(*Command).execute(0xc000381d40, 0x1386488, 0x0, 0x0, 0xc000381d40, 0x1386488)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.0-20171231101556-b95ab734e27d/command.go:750 +0x29d
github.com/spf13/cobra.(*Command).ExecuteC(0xc00039d8c0, 0x1a, 0x1a, 0xc00039d8c0)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.0-20171231101556-b95ab734e27d/command.go:831 +0x2c9
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.0-20171231101556-b95ab734e27d/command.go:784
main.main()
	/tmp/QnVpbGQgYW5kIFBhY2thZ2UgQWxs/run/cli/cdsctl/main.go:50 +0x74d
```

--- a/cli/cdsctl/config.go
+++ b/cli/cdsctl/config.go
@@ -112,6 +112,10 @@ func loadConfig(cmd *cobra.Command) (string, *cdsclient.Config, error) {
 
 	cdsctx.Verbose = verbose
 
+	if cdsctx.Host == "" {
+		return "", nil, fmt.Errorf("invalid cdsctl configuration to reach a CDS API.")
+	}
+
 	config := &cdsclient.Config{
 		Host:                              cdsctx.Host,
 		SessionToken:                      cdsctx.Session,

--- a/cli/cdsctl/main.go
+++ b/cli/cdsctl/main.go
@@ -80,7 +80,7 @@ func rootFromSubCommands(cmds []*cobra.Command) *cobra.Command {
 			return
 		}
 
-		cli.ExitOnError(err, login().Help)
+		cli.ExitOnError(err, root.Help)
 	}
 
 	return root

--- a/cli/cdsctl/version.go
+++ b/cli/cdsctl/version.go
@@ -50,7 +50,7 @@ func versionRun(v cli.Values) error {
 	}
 
 	var versionAPI *sdk.Version
-	if cfg.Host != "" {
+	if cfg != nil && cfg.Host != "" {
 		var err error
 		versionAPI, err = client.Version()
 		if err != nil {

